docs/docs-content/vertex/system-management/account-management/password-blocklist.md

```diff
@@ -0,0 +1,259 @@
+---
+sidebar_label: "Manage Password Blocklist"
+title: "Manage Password Blocklist"
+description: "Learn how to manage the password blocklist in Palette VerteX."
+icon: ""
+hide_table_of_contents: false
+sidebar_position: 50
+tags: ["vertex", "management", "account", "credentials"]
+keywords: ["self-hosted", "vertex"]
+---
+
+You can manage a password blocklist to prevent users from using common or weak passwords. The password blocklist is a
+list of passwords that users cannot use when setting or updating their passwords. The password blocklist is enforced
+when users set or update their passwords.
+
+:::info
+
+Refer to [Password Requirements and Security](credentials.md#password-requirements-and-security) to learn more about the
+password requirements and security in Palette VerteX.
+
+:::
+
+The internal System API is used to manage the password blocklist. Review the following sections to learn how to maintain
+the password blocklist.
+
+## View Passwords in the Blocklist
+
+There is no GET endpoint to retrieve the list of passwords in the blocklist. All passwords are hashed and salted, and
+the cryptographic key of the password is stored in the internal system database. The actual password values in the
+blocklist are not stored in the database. Due to this reason, there is no way to retrieve the plain text list of
```

Comment:
Maybe it is better to be move the last sentence to the beginning of the paragraph telling users upfront that there is no way to to retrieve it and then explain the reason?

Reply:
Good call :)